Extensions/TextObject/Extension.cpp

```diff
@@ -364,6 +428,65 @@ void DeclareTextObjectExtension(gd::PlatformExtension& extension) {
       .SetManipulatedType("number")
       .SetIncludeFile("TextObject/TextObject.h");
 
+  obj.AddCondition("GetPadding",
+                   _("Padding"),
+                   _("Compare the number of pixels around a text object. If you text gets cropped due to shadow effects or an outline raise this value."),
+                   _("The padding of _PARAM0_ is _PARAM1__PARAM2_"),
+                   _("Style"),
+                   "res/conditions/textPadding24.png",
+                   "res/conditions/textPadding.png")
+
+      .AddParameter("object", _("Object"), "Text")
+      .AddParameter("relationalOperator", _("Sign of the test"))
+      .AddParameter("expression", _("Value to test"))
+      .SetFunctionName("GetPadding")
+      .SetManipulatedType("number")
+      .SetIncludeFile("TextObject/TextObject.h");
+
+  obj.AddAction("SetPadding",
+                _("Padding"),
+                _("Set the number of pixels around a text object. If you text gets cropped due to shadow effects or an outline raise this value."),
+                _("Do _PARAM1__PARAM2_ to the padding of _PARAM0_"),
+                _("Style"),
+                "res/actions/textPadding24.png",
+                "res/actions/textPadding.png")
+
+      .AddParameter("object", _("Object"), "Text")
+      .AddParameter("operator", _("Modification's sign"))
+      .AddParameter("expression", _("Value"))
+      .SetManipulatedType("number")
+      .SetFunctionName("SetPadding")
+      .SetIncludeFile("TextObject/TextObject.h");
+
+  obj.AddAction(
+         "SetTextAlignment",
+         _("Alignment"),
+         _("Set the text alignment of a multiline text object. (Does not work with single line texts)"),
+         _("Align _PARAM0_: _PARAM1_"),
+         _("Style"),
+         "res/actions/textAlign24.png",
+         "res/actions/textAlign.png")
+
+      .AddParameter("object", _("Object"), "Text")
+      .AddParameter("stringWithSelector", _("Alignment"), "[\"left\", \"center\", \"right\"]", false)
+      .SetFunctionName("SetTextAlignment")
+      .SetIncludeFile("TextObject/TextObject.h");
+
+  obj.AddCondition("GetTextAlignment",
+                   _("Alignment"),
+                   _("Compare the text alignment of a multiline text object."),
+                   _("The alignment of _PARAM0_ is _PARAM1__PARAM2_"),
+                   _("Style"),
+                   "res/conditions/textAlign24.png",
+                   "res/conditions/textAlign.png")
+
+      .AddParameter("object", _("Object"), "Text")
+      .AddParameter("relationalOperator", _("Sign of the test"))
+      .AddParameter("string", _("Text to test"))
```

Comment:
"Text to compare" seems better English?

Reply:
This was copied right from "Compare the text" ;)